### DISCUSSION TITLE
How to Set / View Export Preferences ..

### DISCUSSION THREAD

**Discussion by soliton37** [ansys/pyaedt#5280] [Q&A]
Hi,
I guess with pyaedt, how to "Set / View Export Preferences" in "Export On Completion" tab of HFSS.
Thank, Emmanuel

**Comment by Samuelopez-ansys** [comment]
Hi, you have a set of two methods to do it. First this one to enable it:

https://aedt.docs.pyansys.com/version/dev/API/_autosummary/ansys.aedt.core.hfss.Hfss.export_touchstone_on_completion.html

And then this one to set the settings:

[set_export_touchstone](https://aedt.docs.pyansys.com/version/dev/API/_autosummary/ansys.aedt.core.hfss.Hfss.set_export_touchstone.html)
